CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.13.1 (02.05.2023)
+* Update `scaddress` based on `strkey`
```

Comment:
I think it could be more descriptive. Also, add "Update all dependencies" to the changelog.